<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigPanelWindow</class>
 <widget class="QWidget" name="ConfigPanelWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>717</width>
    <height>497</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>屏保配置</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>图片位置:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="inputDirEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="chooseDirBtn">
          <property name="text">
           <string>选择位置</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>切换速度:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="sliderSpeed">
          <property name="minimum">
           <number>100</number>
          </property>
          <property name="maximum">
           <number>6000</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="sliderSpeedLab">
          <property name="text">
           <string>当前速度: 0.1 秒</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="timerVisible">
     <property name="focusPolicy">
      <enum>Qt::WheelFocus</enum>
     </property>
     <property name="title">
      <string>时钟设置(显示时钟)</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>时钟位置:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comb">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <item>
           <property name="text">
            <string>左上</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>中上</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>右上</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>左</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>中</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>右</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>左下</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>中下</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>右下</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>时钟大小:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="sliderSize">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>300</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="colorBtn">
          <property name="styleSheet">
           <string notr="true">color: #fff</string>
          </property>
          <property name="text">
           <string>颜色</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>时钟阴影:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="sliderShadow">
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>阴影厚度:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="sliderShadowRadius">
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="previewBox">
     <property name="title">
      <string>屏保预览</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3"/>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
